mypyc/codegen/emit.py

```diff
@@ -191,10 +193,25 @@ def reg(self, reg: Value) -> str:
     def attr(self, name: str) -> str:
         return ATTR_PREFIX + name
 
-    def emit_line(self, line: str = "") -> None:
+    def emit_line(self, line: str = "", *, ann: object = None) -> None:
         if line.startswith("}"):
             self.dedent()
-        self.fragments.append(self._indent * " " + line + "\n")
+
+        comment = comment_continued = ""
+        if ann is not None:
+            line_width = self._indent + len(line)
```

Comment:
Can you move this section of code to a helper function, and write a few unit tests?

Reply:
https://github.com/python/mypy/pull/14917/commits/d36e1150285ff7799ade252be619ab302fad1478.